crates/db_views_moderator/src/mod_remove_comment_view.rs

```diff
@@ -58,6 +58,11 @@ impl ModRemoveCommentView {
       query = query.filter(comment::id.eq(comment_id));
     }
 
+    // If a post ID is given, then don't find any results
+    if params.post_id.is_some() {
```

Comment:
This one's the only weird case. We *could* show comment removals (since this has to join to the post table anyway, and filter by post_id) . 

I tried it out in the front end, and although it works, it seems like if you provide a post id, you only want to see mod actions done on the post, not comments within the post. 

I could be persuaded otherwise tho, as it might be useful to see actions taken on comments when filtering by a post id.

Reply:
I think the approach you already made is fine.

Reply:
Both options should be possible, so there should be a "show_children" option that includes comments when given post_id, child comments when given comment_id, and maybe posts and comments when given community_id

Reply:
Not a bad idea. Could you open an issue for that one? Lets leave that for a later time.

Reply:
I think its fine like this.